### ISSUE TITLE
Samba de Amigo Controller Support

### ISSUE THREAD

**Issue by Tobias43** [reicast/reicast-emulator#1464]
I am a huge fan of Samba de Amigo and would love to be able to play it with my Maracas on Reicast.

I could get a Dreamcast controller to USB adapter, but I suppose that that is not enough and there has to be some kind of software support for the Maraca controller in Reicast, too.

**Comment by dmiller423** [comment]
Yes,  it would require emulation of the maple device,  and i'm not sure which usb adapter might support the particular device.  It might be that it will support them as the analogs,  in a hid gamepad or not be supported.   I have one usb adapter here but no maracas,  so I can't be of much help there.

**Comment by Tobias43** [comment]
Hi David, i have two sets of Maracas (non-original ones, but they work well). If you want to give the emulation a try, and the shipping from Germany is not prohibitively expensive, i will send you one set (you can keep it).

**Comment by dmiller423** [comment]
Don't they require a separate interface panel and some sensors?
I'd be happy to check if the usb device I have supports them,  and add the maple device to the emulator,  but I have no way to guarantee it will work.  It might be simple to get the wii controller maraca adapters and get them working via bluetooth:  just a thought.
Anyhow if you want,  I will test them out and let you know.  The shipping address would be to US, Indiana

**Comment by Tobias43** [comment]
Hi David, I finally took the Maracas down from my parents attic. They are light enough to ship them for a reasonable price. So if you PM me your address, they are on the way, including the sensor panel. Thank you very much in advance! I think you are going to like the game!

I already tried Samba de Amigo on the Wii, but it is not the same. As the Wiimotes only track orientation and movement, but not their absolute position, they don't work well at all for the game.
